src/flattenizer.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Flattens an object
+ *
+ * @param {Object} obj - the object to unflatten
+ * @returns {Object} - the flattened object, empty if provided object is undefined
+ */
+export const flatten = (obj) => {
+   let flattened = {};
+
+    for (let key in obj) {
+        if (!obj.hasOwnProperty(key))
+            continue;
+
+        if (typeof obj[key] === 'object') {
+            let flatObject = flatten(obj[key]);
+            for (let subKey in flatObject) {
+                // append to create new key value and assign it's value
+                flattened[key + '.' + subKey] = flatObject[subKey];
+            }
+        } else {
+            flattened[key] = obj[key];
+        }
+    }
+
+    return flattened;
+};
+
+/**
+ * Unflattens an object with compressed keys.
+ *
+ * @param {Object} flattened - object to unflatten
+ * @returns {Object} - the unflattened object, empty if provided object is undefined
+ */
+export const unflatten = flattened => {
```

Comment:
You ought to wrap your params consistently:

* single value to single return === no parentheses or braces
* single value to code block === parentheses and braces
* multiple value to code block === parentheses and braces
* multiple value to single return === parentheses and no braces